PicApp_git/PicApp/routes/web.php

```diff
@@ -20,4 +20,10 @@
     Route::post('/top', 'PostsController@store');
```

Comment:
今回の差分じゃないですが、ここは投稿のルーティングなので下記が良さそうです
```suggestion
    Route::post('/posts', 'PostsController@store');
```